school/decorate.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name(name)
+    name.capitalize![0..9]
```

Comment:
- [ ] From the project requirements, this method is supposed to rails `NotImplementedError`
![image](https://user-images.githubusercontent.com/90222110/194086831-eb681a56-3061-4297-aec7-67446cd438c5.png)
Please correct it 😃 🏃